src/java/htsjdk/samtools/metrics/MetricsFile.java

```diff
@@ -84,6 +84,13 @@ public void addAllMetrics(final Iterable<BEAN> beanz) {
         else return null;
     }
 
+    /** Returns the histogram contained in the metrics file if any. Adds capability of retrieving multiple histograms
+     *  from metrics file. */
+    public Histogram<HKEY> getHistogram(final int index) {
```

Comment:
Thank you Louis - Do you mean to do just a try return statement and catch and throw an exception instead of checking in the if and returning null in the 'else' ?  

I'm also working on adding tests because Yossi pointed out I need them - there's no tests for either getHistogram function so I'm doing that. So this PR will need to wait a little for that.